include/alpaka/acc/AccCpuThreads.hpp

```diff
@@ -108,7 +113,11 @@ namespace alpaka
                     workdiv::WorkDivMembers<TDim, TSize>(workDiv),
                     idx::gb::IdxGbRef<TDim, TSize>(m_gridBlockIdx),
                     idx::bt::IdxBtRefThreadIdMap<TDim, TSize>(m_threadToIndexMap),
-                    atomic::AtomicStlLock(),
+                    atomic::AtomicHierarchy<
+                        atomic::AtomicStlLock,    // atomics between grids
```

Comment:
At the moment I like the comments, because it explains the hierarchy to me. However, it should be very well documented within the `AtomicHierarchy` class.